CONTRIBUTING.md

```diff
@@ -9,7 +9,10 @@ To start the local site use:
 To deploy updated versions:
 `npm run deploy`
 
+# Using Cloudflare Workers
 
+1. Run `npx wrangler dev src/index.js` in your terminal to start a development server
+1. Open a browser tab at http://localhost:8080/ to see your worker in action
+1. Run `npx wrangler publish src/index.js --name my-worker` to publish your worker
 
```

Comment:
Make the script generic: `npx wrangler publish <file-path> --name <worker-name>`